internal/controller/controllers/agentserviceconfig_controller.go

```diff
@@ -519,6 +521,24 @@ func (r *AgentServiceConfigReconciler) newPostgresSecret(instance *aiv1beta1.Age
 }
 
 func (r *AgentServiceConfigReconciler) newAssistedServiceDeployment(instance *aiv1beta1.AgentServiceConfig, serviceURL *url.URL) (*appsv1.Deployment, controllerutil.MutateFn) {
+
+	// User is responsible for knowing to restart assisted-service
+	var envFrom []corev1.EnvFromSource
+	annotations := instance.ObjectMeta.GetAnnotations()
+	configmapName, ok := annotations[configmapAnnotation]
+	if ok {
+		r.Log.Info("ConfigMap %v being used to configure assisted-service deployment")
```

Comment:
Better ? 
`r.Log.Infof("ConfigMap %v being used to configure assisted-service deployment", configmapName)`